### PR TITLE
AutoRest based Key Vault management and tests

### PR DESCRIPTION
**Pull request by pomortaz**
The Key Vault management code is generated according to [this swagger spec](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-keyvault/2015-06-01/swagger/keyvault.json).
The project is renamed from KeyVaultManagement to Microsoft.Azure.Management.KeyVault

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pomortaz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pomortaz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: global.json [hunk 1]
```diff
@@ -22,5 +22,6 @@
     "src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager",
     "src/ResourceManagement/Websites/Microsoft.Azure.Management.Websites",
     "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.DataLake.StoreUploader"
+    "src/ResourceManagement/KeyVaultManagement/Microsoft.Azure.Management.KeyVault"
   ]
```
Comment (hovsepm) [inline global.json]: Please put this in the alphabetical order

File: src/ResourceManagement/KeyVaultManagement/KeyVaultManagement.Tests/VaultOperationsTest.cs [hunk 2]
```diff
@@ -1,14 +1,13 @@
-﻿using System;
-using System.Linq;
+﻿using System.Linq;
 using System.Collections.Generic;
```
Comment (hovsepm) [inline src/ResourceManagement/KeyVaultManagement/KeyVaultManagement.Tests/VaultOperationsTest.cs]: License header missing for this file

File: src/ResourceManagement/KeyVaultManagement/Microsoft.Azure.Management.KeyVault/project.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  "description": "Provides resource group and resource management capabilities for Microsoft Azure.",
+  "authors": [ "Microsoft" ],
+
+  "packOptions": {
+    "projectUrl": "https://github.com/Azure/azure-sdk-for-net",
+    "licenseUrl": "https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE",
+    "tags": [ "Microsoft Azure key vault management", "Key Vault", "REST HTTP client", "azureofficial", "windowsazureofficial" ],
+    "requireLicenseAcceptance": true,
+    "releaseNotes": "This is a preview release of the Azure Key Vault .NET SDK, based on version 2015-06-01 of the Azure Key Vault REST API."
+  },
```
Comment (hovsepm) [inline src/ResourceManagement/KeyVaultManagement/Microsoft.Azure.Management.KeyVault/project.json]: please add here: "iconUrl": "[http://go.microsoft.com/fwlink/?LinkID=288890](http://go.microsoft.com/fwlink/?LinkID=288890)",

**Comment by hovsepm** [comment]
@pomortaz please address review comments.

**Comment by hovsepm** [comment]
@pomortaz please add AssemblyInfo.cs file for the project.

**Comment by pomortaz** [comment]
@hovsepm thanks. I added the AssemblyInfo.cs

**Pull request merged**